tests/mlmodel_openai/conftest.py

```diff
@@ -65,7 +65,6 @@
         "test_chat_completion.py",
         "test_get_llm_message_ids.py",
         "test_chat_completion_error.py",
```

Comment:
I made a commit to remove this line. I had added it for my PR since these tests were failing in my branch. I think because the tests weren't actually running before and now they are there's some errors that need to be fixed in them.